sos/report/plugins/authd.py

```diff
@@ -0,0 +1,53 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+# See the docs: https://documentation.ubuntu.com/authd/
+class Authd(Plugin, UbuntuPlugin):
```

Comment:
For the `authd` plugin do we need to check for any service status, or journal logs that could be useful?

Reply:
Is it idomatic for sos to assume that version information, systemd unit status & logs will be collected by the `snap`, `apt`, and `systemd` plugins rather than the plugin for a specific service's configuration? Happy to add those things if needed.

The only information that I've not been able to view about authd while working cases has been the content of these config files. The [authd bug report template](https://github.com/ubuntu/authd-oidc-brokers/issues/new?template=bug_report.yml) has a reasonable collection of information to grab and I'm quite confident that with this plugin all of that will be included in a sosreport.

Reply:
We also have to assume that if this plugin is ran standalone with `-o authd`, then it could be that this information is not collected, imo, we should assume that this is a standalone plugin, so if we need to collect those then let's get them added

Reply:
Added systemd services and an authd version check to the plugin; they automatically call [`add_service_status`](https://sos.readthedocs.io/en/main/plugins.html#sos.report.plugins.Plugin.add_service_status) and [`add_journal`](https://sos.readthedocs.io/en/main/plugins.html#sos.report.plugins.Plugin.add_journal) for those services.